assets/src/edit-story/components/colorPicker/editablePreview.js

```diff
@@ -32,7 +32,6 @@ import {
 } from '../../../design-system';
 
 const Preview = styled.button`
-  padding: 0;
```

Comment:
Hm, how is this related 🤔, from some merge? It should have the `0`, otherwise the preview will look different from the edit mode.

Reply:
Yes, it was from a (bad) merge. 0 is the right one?

Reply:
Yes, 0 is the right one.